File: android/app/src/main/java/com/exitapp/ExitModule.java
package com.exitapp;
 
import androidx.annotation.NonNull;
 
import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactContextBaseJavaModule;
import com.facebook.react.bridge.ReactMethod;
 
public class ExitModule extends ReactContextBaseJavaModule {
 
   private static ReactApplicationContext reactContext;
 
   public ExitModule(ReactApplicationContext reactContext) {
       super(reactContext);
       this.reactContext = reactContext;
   }
 
   @ReactMethod
   public void exit() {
       android.os.Process.killProcess(android.os.Process.myPid());
   }
 
   /* Here create multiple ReactMethod */
 
   @NonNull
   @Override
   public String getName() {
       return "ExitApp";
   }
}

File: ios/ExitApp/ExitModule.m
//
//  ExitModule.m
//  ExitApp
//
//  Created by Mahesh Bokhani on 09/07/20.

#import "ExitModule.h"
 
@implementation ExitApp
 
RCT_EXPORT_MODULE()
 
RCT_EXPORT_METHOD(exit)
{
 NSLog(@"Exit Button Press");
 exit(0);
}
// Here create multiple RCT_EXPORT_METHOD
 
@end
